domains/parth.json

```diff
@@ -0,0 +1,13 @@
+{
+    "owner": {
+        "username": "TheInfernitex",
+        "email": "agarwalparthsplendid@gmail.com"
+    },
+    "target": {
+        "CNAME": {
+            "name": "*.parth",
```

Comment:
We do not support wildcards, you must register each individual nested subdomain.

Reply:
oh. okay, I'll change it.